src/build/__tests__/build-test.js

```diff
@@ -149,4 +168,16 @@ describe('build()', () => {
       Timestamp: { serialize },
     }, undefined, false)).forEach((type, i) => expectTypesEqual(type, [Record, Timestamp][i]));
   });
+
+  it('should allow directives configuration using __schema', () => {
+    const config = { __schema: { directives: [customDirective] } };
+    const target = build(schemaSource + querySourceWithDirective, config, undefined, false);
+    expectSchemasEqual(target, SchemaWithDirective);
+  });
+
+  it('should not build schema with default directives', () => {
```

Comment:
- **'should allow directives configuration using __schema'** tests for to see that we can pass __schema.
- **'should not build schema with default directives'** tests that the schema returned by build is not same when config __schema is passed. This extra test was added because I had one mistake in my setup code but the first test (i.e: 'should allow directives configuration using __schema') didn't catch that mistake.